bin/elixirc.bat

```diff
@@ -15,11 +15,14 @@ goto run
 echo Usage: %~nx0 [elixir switches] [compiler switches] [.ex files]
 echo.
 echo   -o                          The directory to output compiled files
-echo   --no-docs                   Does not attach documentation to compiled modules
+echo.
+echo   --help, -h                  Prints this message and exits
+echo   --ignore-module-conflict    Does not emit warnings if a module was previously defined
 echo   --no-debug-info             Does not attach debug info to compiled modules
+echo   --no-docs                   Does not attach documentation to compiled modules
 echo   --verbose                   Prints compilation status
+echo   --version, -v               Prints Elixir version and exits
 echo   --warnings-as-errors        Treats warnings as errors and returns non-zero exit code
-echo   --ignore-module-conflict    Does not emit warnings if a module was previously defined
 echo.
 echo ** Options given after -- are passed down to the executed code
 echo ** Options can be passed to the Erlang runtime using ELIXIR_ERL_OPTIONS
```

Comment:
It is mentioned in the header under [elixir switches], we can make it clearer if necessary but I wouldn't copy them over.